compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc

```diff
@@ -47,8 +47,6 @@ For the on-demand scan, you must pre-define the image registry scope in the regi
 
 Verify that the images in the repository are being scanned.
 
-NOTE: From 22.11 (Lagrange) release or later, you can add a maximum of 19,999 registry entries in **Defend > Vulnerabilities > Images > Registry settings**. 
-
 [.procedure]
 . Go to *Monitor > Vulnerabilities > Images > Registries*.
```

Comment:
Have you added the limits for 19,999 registries ( Defend > Vuln > Images > Registry Settings)  and  up to 100,000 images (Monitor > Vulnerabilities > Images > Registries) on the Limits topic?

Reply:
@Pubs-MV Yes in line no: 20